.gitignore

```
@@ -127,3 +127,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# others
+*.json
+*.ipynb
```

better to add yaml file to git ignore
```suggestion
*.ipynb
*.yaml
```